api/all.yaml

```diff
@@ -21,6 +21,14 @@ tags:
       Functions implement User-Defined Functions (UDFs) in Unity Catalog.
       The function implementation can be a SQL expression/query or a python function, and it can be invoked wherever a table reference is allowed in a query.
       In Unity Catalog, a function resides at the same level as a table, so it can be referenced with the form __catalog_name__.__schema_name__.__function_name__.
+  - name: Models
+    description: |
+      Models are machine learned models that are stored within Unity Catalog.  They are comprised of all versions of a single model that a user wishes to keep.
```

Comment:
"machine learned"? Do we want docs team to do a quick pass here as well?

Reply:
Probably too early for any of that.  We can fix the documentation up later.  I don't want to get too hung up or married to any documentation right now.